Codes/edit.php

```diff
@@ -87,17 +67,6 @@
       <div id="content">
           
         <main class="col-md-auto ms-sm-3 col-lg-auto px-md-auto">
-        <?php
-        if (isset($_GET['course']))
-        {
-          $cd = $_GET['course'];
-        }
-        else  
-        {
-          header('Location: dashboard.php?error=ERROR ');
-          exit();
-        }
-        ?>
```

Comment:
any feasible explanation for removing this part

Reply:
This part is repeated here. --line 28